Druid/DRAbstractMemoryAddress.class.st

```diff
@@ -99,3 +99,9 @@ DRAbstractMemoryAddress >> rtlPushOperandQualifierForInstruction: aPush [
 DRAbstractMemoryAddress >> type [
 	^ type
 ]
+
+{ #category : #accessing }
+DRAbstractMemoryAddress >> type: aDRType [
+
+	type := aDRType
```

Comment:
An address has a type?

Reply:
That seams :P

I don't remember, I had broken tests.

Anyway, as an address could be, for example, an interpreter reference, that could be an object pointer (like `trueObj`) or an address (like `oldSpaceStart`).